docs/index.html

```diff
@@ -786,7 +786,7 @@ <h3 id="data-points-singular-block">Data Points - Singular - Block</h3><a class=
 
 <h4>Expected Contexts</h3>
 <ul>
-<li><span class="distinct">article.case-studies > ul > li > article-data-point</span></li>
+<li><span class="distinct">article.case-studies > ul > li > article.data-point</span></li>
```

Comment:
unsure why this change is included?

Reply:
HI @possumbilities  I am working on it now.